src/Peachpie.Library/Strings.cs

```diff
@@ -37,6 +37,11 @@ internal static CharMap InitializeCharMap()
 
         #endregion
 
+        // Helper function to change Console.OutputEncoding from PHP
+        public static void peachpie_set_string_encoding(string str) {
```

Comment:
We can't just add a public function to the API like that. You can always define this method in privately your project, or we could add a configuration option for this.

Reply:
> We can't just add a public function to the API like that. You can always define this method in privately your project, or we could add a configuration option for this.

The configuration options sounds good to me.